NOnoN0nOYes.js

```diff
@@ -0,0 +1,58 @@
+//Save a note
+
+const notes = [];
+
+function saveNote(content, id) {
+  const note = {
+    content: content,
+    id: id,
+  };
+  notes.push(note);
+}
+saveNote("Pick up groceries", 2);
+saveNote("Take a walk", 5);
+console.log(notes);
+
+//Get a note
+
+function getNote(id) {
+  if (id === undefined || typeof id !== "number") {
```

Comment:
Small tip: the check for `undefined` here can be safely omitted. The thing is that when you call `typeof a` for `a` which is undefined, you get `"undefined"`:

```js
let a = undefined;
typeof a; // "undefined"
```

So you can safely write:
```suggestion
  if (typeof id !== "number") {
```